RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronSeedProducer.cc

```diff
@@ -535,8 +589,13 @@ void LowPtGsfElectronSeedProducer::fillDescriptions( edm::ConfigurationDescripti
   desc.add< std::vector<std::string> >("ModelWeights",std::vector<std::string>());
   desc.add< std::vector<double> >("ModelThresholds",std::vector<double>());
   desc.add<bool>("PassThrough",false);
-  desc.add<bool>("UsePfTracks",false);
-  desc.add<double>("MinPtThreshold",0.5);
+  desc.add<bool>("UsePfTracks",true);
+  desc.add<double>("MinPtThreshold",1.0);
```

Comment:
This higher "MinPtThreshold" tightens even more the working point of the default config, which is visible in some reduced CPU time of the lowPt gsf electrons and in the correspondign output size. (This is just an observation, in order to keep note of it.)